src/GetAssemblyInfoViz.cpp

```diff
@@ -0,0 +1,196 @@
+
+
+#include "GetAssemblyInfoViz.h"
+
+using namespace std;
```

Comment:
In general, don't use `using namespace` in Halide source at all, at least inside the `src/` directory. This code should probably all be in the `Halide::Internal` namespace anyway, so the usings would be redundant.